packages/core/src/types/config/access-control.ts

```diff
@@ -89,22 +84,22 @@ type ListAccessControlObject<ListTypeInfo extends BaseListTypeInfo> = {
   // - a filter. In this case, the operation can proceed, but the filter will be additionally applied when updating/reading/deleting
   //   which may make it appear that some of the items don't exist.
   // - boolean true/false. If false, treated as a filter that never matches.
-  filter?: {
-    query?: ListFilterAccessControl<'query', ListTypeInfo>;
-    update?: ListFilterAccessControl<'update', ListTypeInfo>;
-    delete?: ListFilterAccessControl<'delete', ListTypeInfo>;
-    // create: not supported: FIXME: Add explicit check that people don't try this.
-    // FIXME: Write tests for parseAccessControl.
-  };
+  filter?:
+    | ListFilterAccessControl<FilterOperation, ListTypeInfo>
```

Comment:
This may not be desirable, but, it is what the code currently supports

https://github.com/keystonejs/keystone/blob/c3a3789c6e4a1761a24cd02904b7549a62113363/packages/core/src/lib/core/access-control.ts#L149

Reply:
We shouldn't support this

Reply:
https://github.com/keystonejs/keystone/pull/7914/commits/51022994db76e68a4d90979cb48ae1eb065e73ff